draft-ietf-quic-http.md

```diff
@@ -402,22 +402,23 @@ All frames have the following format:
  0                   1                   2                   3
  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|                           Length (i)                        ...
+|    Type (8)   |               Length (i)                    ...
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|    Type (8)   |               Frame Payload (*)             ...
+|                     Frame Payload (*)                       ...
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 ~~~~~~~~~~
 {: #fig-frame title="HTTP/3 frame format"}
 
 A frame includes the following fields:
 
+  Type:
+  : An 8-bit type for the frame.
+
+
   Length:
   : A variable-length integer that describes the length of the Frame Payload.
     This length does not include the Type field.
 
```

Comment:
With reordering the "Length" description no longer makes sense. If anything, length should state that it does not include itself.